pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -79,6 +79,8 @@ message DNSOption {
     repeated string servers = 1;
     // List of DNS search domains of the cluster.
     repeated string searches = 2;
+    // List of DNS opt strings
+    repeated string opts = 3;
```

Comment:
@yujuhong I don't think it should be called ndots, ndots is just one available options. And it's not all key-value pairs. See https://linux.die.net/man/5/resolv.conf

The canonical name is just options, it's unfortunate that we've already used that as the name of the whole struct.